### PR TITLE
Added support for debugging multi-process tasks in VSCode.

### PR DESCRIPTION
**Pull request by davidhopper2003**
Refer to my issue [Unable to debug the PPO2 algorithm with VSCode](https://github.com/openai/baselines/issues/739).

### CODE REVIEW

**Comment by ashrafulislamM** [comment]
Thanks

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
+{
```
Comment (pzhokhov) [inline .vscode/launch.json]: I don't use VSCode so cannot really review / test this file. However, it seems like there are some run-specific arguments embedded into it. If we were to include it, I'd prefer a more generic version (maybe instead a doc describing how to generate this file)

**Comment by pzhokhov** [comment]
@davidhopper2003 while it looks like a sensible change, could you look into fixing the failing travis tests for this PR? Looks like there a few and they seem to be related to the change. Please post here if you need some help with that.

**Comment by pzhokhov** [comment]
closing this (upstream is changed to using method "spawn")

**Pull request closed without merging**